src/app/recipe-details/recipe-details.component.html

```diff
@@ -1,11 +1,11 @@
-<!DOCTYPE html>
+<!-- <!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
-</head>
-<app-nav-bar></app-nav-bar>
+</head> -->
+<!-- <app-nav-bar></app-nav-bar> -->
 <br/>
 <body class="bg">
```

Comment:
I don't think we need the body tags either, this is similar to the HTML tags at the top.